### DISCUSSION TITLE
How to use ui5-webcomponents-react with NextJS

### DISCUSSION THREAD

**Discussion by softshipper** [UI5/webcomponents-react#2961] [Q&A] [answered]
Hi all,

I would like to use ui5-webcomponents-react in NextJS and would like to know if additional configurations are needed. 

Thanks

**Comment by MarcusNotheis** [comment, marked as answer]
Hi @softshipper,
sorry for the late reply, we somehow missed this discussion.
We're currently working on support Server Side Rendering frameworks like Next.js, you can track the progress via #4091.
